src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int maxDiggingDepth;
+    private int armLength;
```

Comment:
The field 'armLength' should have an access modifier. Consider using private to encapsulate the field properly.